raiden/transfer/channel.py

```diff
@@ -389,6 +406,146 @@ def is_valid_lockedtransfer(
     )
 
 
+def is_valid_lockexpired(
```

Comment:
please let's be consistent: ` is_valid_lockexpired` -> ` is_valid_lock_expired`

just like most of the other functions in this file.